common/namespace/namespace.go

```diff
@@ -290,3 +309,27 @@ func (n Name) String() string {
 func (n Name) IsEmpty() bool {
 	return n == EmptyName
 }
+
+func (m *CustomSearchAttributesMapper) GetAlias(fieldName string, namespace string) (string, error) {
+	alias, ok := m.fieldToAlias[fieldName]
+	if !ok {
+		return "", serviceerror.NewInvalidArgument(
+			fmt.Sprintf("Namespace %s has no mapping defined for field name %s", namespace, fieldName),
+		)
+	}
+	return alias, nil
+}
+
+func (m *CustomSearchAttributesMapper) GetFieldName(alias string, namespace string) (string, error) {
+	fieldName, ok := m.aliasToField[alias]
+	if !ok {
+		return "", serviceerror.NewInvalidArgument(
+			fmt.Sprintf("Namespace %s has no mapping defined for search attribute %s", namespace, fieldName),
+		)
+	}
+	return fieldName, nil
+}
```

Comment:
1. We should change the return type to (string, bool) because these are utility functions and the service-level functions exist higher up in the call stack
2. We should remove the namespace parameter because it is not used for functionality here. Callers can dictate what the error should be and if a namespace is attached to it.

Reply:
We need the namespace because it's part of the `searchattribute.Mapper` interface.
Maybe in another follow up PR to remove it.

Reply:
Ah, I see